src/RemoteSymbolProvider/include/RemoteSymbolProvider/DownloadManagerInterface.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef REMOTE_SYMBOL_PROVIDER_DOWNLOAD_MANAGER_INTERFACE_H_
+#define REMOTE_SYMBOL_PROVIDER_DOWNLOAD_MANAGER_INTERFACE_H_
+
+#include "OrbitBase/CanceledOr.h"
+#include "OrbitBase/Result.h"
+#include "OrbitBase/StopToken.h"
+
+namespace orbit_remote_symbol_provider {
```

Comment:
I think this interface should be in the `Http` module because it's not related to the Symbol Provider at all. What was your reasoning in putting it here? Maybe I don't see all the constraints?

Reply:
Done! Thanks!